changelog.d/6833.misc

```diff
@@ -0,0 +1 @@
+Reducing log level to DEBUG for synapse.storage.TIME.
```

Comment:
quick reminder that changelogs should be in the imperative mood: `Reduce log level to DEBUG for synapse.storage.TIME.`